### PR TITLE
115 slice position task needs a folder of dicoms

### PR DESCRIPTION
**Pull request by Lucrezia-Cester**
I have added a functionality that allows a folder to be uploaded. The DICOM headers are stored correctly on the database. However I still need to figure out how to display the folder on the webpage.

### CODE REVIEW

**Comment by Lucrezia-Cester** [comment]
This code allows the user to upload a folder and click on the folder as one of the available files. I am currently stuck at database migration.

**Comment by Lucrezia-Cester** [comment]
[

https://user-images.githubusercontent.com/83493021/230040533-48333bc8-b6f2-47f6-b5e9-c8ceb92f2bd0.mov

](url)

Slice Position requires a folder as input. 

I have created a function upload_folder() function handles the folder upload process. **In theory,** it should saves the uploaded folder to the server, extracts the files from the ZIP archive, and then calls the ingest() function for each file in the folder.

However I checked in postico and the database does not update when the folder is uploaded. 

@hshuaib90

File: migrations_new/alembic.ini
Comment (heyhaleema) [inline migrations_new/alembic.ini:1]: @Lucrezia-Cester Similar to [this](https://github.com/GSTT-CSC/hazen-web-app/pull/103#discussion_r1170160032) comment in #103, I just want to double-check whether this `migrations_new` folder is should be pushed to the repo?

File: migrations/env.py
Comment (heyhaleema) [inline migrations/env.py:1]: @Lucrezia-Cester Similar to [this](https://github.com/GSTT-CSC/hazen-web-app/pull/103#discussion_r1170160032) comment in #103, I just want to double-check whether this `migrations` folder is should be pushed to the repo?

File: app/__init__.py [hunk 1]
```diff
@@ -32,67 +21,33 @@
...
     heroku.init_app(app)
 
-    # Add project-specific functionality
-    # Main page - Dashboard
-    from app.main import bp as main_bp
-    app.register_blueprint(main_bp)
-    # Upload directory to hold files
-    os.makedirs(app.config['UPLOADED_PATH'], exist_ok=True)
+    # Set SQLALCHEMY_DATABASE_URI to PostgreSQL URL string
+    app.config['SQLALCHEMY_DATABASE_URI'] = f"postgresql://{os.environ['POSTGRES_USER']}@localhost:5432/{os.environ['POSTGRES_DB']}"
```
Comment (heyhaleema) [inline app/__init__.py:40]: @Lucrezia-Cester I've just tried to run the app and I receive the following error:

image

What is the reasoning behind this change? I'm wondering if I've missed a setup step that's causing it to fail...

File: migrations_new/alembic.ini
Comment (hshuaib90) [reply migrations_new/alembic.ini:1]: There should only be one migrations folder.

**Comment by sophie22** [comment]
The slice position task expects 60 input files, which can be selected as a series, having uploaded all 60 files from a folder/series using the existing upload function. The slice position task can then be run on this series of 60 files, without further modifications to the code.

**Pull request closed without merging**